### PR TITLE
Helitron Scanner Draw

### PR DESCRIPTION
**Pull request by jguhlin**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.

Helitron scanner PairEnds & Draw command

### CODE REVIEW

**Comment by jguhlin** [comment]
There's one last step for EDTA after this, which merges the forward and reverse runs.
https://github.com/oushujun/EDTA/blob/c745fd36fadcbf63eddecc827695b1c8d0cd56b0/util/run_helitron_scanner.sh#L64C1-
L64C109

Probably should be in EDTA nextflow script?

**Comment by GallVp** [comment]
I have fixed the linting issues. To write the tests we need to first merge https://github.com/GallVp/nxf-components/pull/41 so that we can use that t generate `head` and `tail` files.

File: modules/gallvp/helitronscanner/draw/main.nf [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        'https://depot.galaxyproject.org/singularity/helitronscanner:1.0--hdfd78af_0':
+        'biocontainers/helitronscanner:1.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    tuple val(meta2), path(head)
+    tuple val(meta3), path(tail)
+
+    output:
+    tuple val(meta), path("*.draw*")    , emit: draw
```
Comment (GallVp) [inline modules/gallvp/helitronscanner/draw/main.nf:16]: I dont' think we need `*.draw*`. `*.draw` should do the job.

File: modules/gallvp/helitronscanner/draw/tests/tags.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+helitronscanner/draw:
```
Comment (GallVp) [inline modules/gallvp/helitronscanner/draw/tests/tags.yml:1]: We should delete this file.

File: modules/gallvp/helitronscanner/draw/main.nf (hunk 1 above)
Comment (GallVp) [reply modules/gallvp/helitronscanner/draw/main.nf]: This has been resolved as the output file was `.draw.hel.fa` which is now moved to `.draw`

File: modules/gallvp/helitronscanner/draw/tests/tags.yml (hunk 2 above)
Comment (GallVp) [reply modules/gallvp/helitronscanner/draw/tests/tags.yml]: Deleted!

**Comment by GallVp** [comment]
Hi @jguhlin

Hopefully all the tests will pass now. Please have a final look and merge it.

**Comment by GallVp** [comment]
Hi @oushujun, @FayeFang17

Hello!

Tagging you here so that you can get an update about our progress on Nextflow/EDTA. You can see that we are porting each module one by one and adding unit tests along the way. The progress is very slow but hopefully we will have a very solid foundation to build upon once the underlying modules and sub workflows have been ported.

**Pull request merged**